packages/appeals-service-api/src/routes/v2/appeal-cases/_caseReference/lpa-questionnaire-submission/documentUpload/repo.js

```diff
@@ -0,0 +1,67 @@
+const { createPrismaClient } = require('#db-client');
+// const { PrismaClientKnownRequestError } = require('@prisma/client/runtime/library');
+
+/**
+ * @typedef {import('@prisma/client').SubmissionDocumentUpload} SubmissionDocumentUpload
+ */
+
+class SubmissionDocumentUploadRepository {
+	dbClient;
+
+	constructor() {
+		this.dbClient = createPrismaClient();
+	}
+
+	/**
+	 * Get all submission documents for given questionnaire
+	 *
+	 * @param {string} questionnaireId
+	 * @returns {Promise<SubmissionDocumentUpload[]|null>}
+	 */
+
+	async getDocumentUploads(questionnaireId) {
+		return await this.dbClient.submissionDocumentUpload.findMany({
+			where: {
+				questionnaireId
+			}
+		});
+	}
+
+	/**
+	 * Create submission document for given questionnaire
+	 *
+	 * @param {object} uploadData
+	 * @returns {Promise<SubmissionDocumentUpload|null>}
```

Comment:
Why `|null`?

Reply:
Good point, only returns null on a `find` so have removed from the create